pkg/server/cstorvolumeconfig/backup_endpoint.go

```diff
@@ -199,7 +199,7 @@ func (bOps *backupAPIOps) get() (interface{}, error) {
 
 	// backup name is expected
 	if len(strings.TrimSpace(backup.Spec.BackupName)) == 0 {
-		return nil, CodedError(400, fmt.Sprintf("Failed to get backup: missing backup name "))
+		return nil, CodedError(400, string("Failed to get backup: missing backup name "))
```

Comment:
We no need to explicitly type cast using `string`. It can be as simple like below
```suggestion
		return nil, CodedError(400, "Failed to get backup: missing backup name ")
```